c2corg_ui/static/partials/contexthelp/route-equipement_rating.html

```diff
@@ -0,0 +1,28 @@
+<p><strong translate>Protection</strong></p>
```

Comment:
But do you really need those subtitles? The window is already entitled using the ``context-help-title`` provided in the forms. Then adding subtitles in the partials seem redundant:

<img width="619" alt="capture d ecran 2017-06-14 a 23 17 48" src="https://user-images.githubusercontent.com/1192331/27155120-f4ec1fea-5157-11e7-954e-70631a42616b.png">